libs/wingsdk/src/platform/parameter-registrar.ts

```diff
@@ -29,7 +29,7 @@ export class ParameterRegistrar extends Construct {
    * @param path the path of the parameter
    * @returns the value of the parameter
    */
-  public getParameterValue(path: string): any {
+  public getValue(path: string): any {
```

Comment:
```suggestion
  public value(path: string): any {
```

We are normally not using `get` this is not Java here